Ginger/GingerCoreNET/Logger/WebReportGenerator.cs

```diff
@@ -46,22 +48,52 @@ public WebReportGenerator(string browserNewPath)
         }
 
         // TODO: Make this function to just generate the report folder !!!
-        public LiteDbRunSet RunNewHtmlReport(string runSetGuid = null, WebReportFilter openObject = null, bool shouldDisplayReport = true)
+        public LiteDbRunSet RunNewHtmlReport(string reportResultsFolderPath = "", string runSetGuid = null, WebReportFilter openObject = null, bool shouldDisplayReport = true)
         {
+            //Copy folder to reportResultsFolderPath or Execution logger
+            HTMLReportsConfiguration currentConf = WorkSpace.Instance.Solution.HTMLReportsConfigurationSetList.Where(x => (x.IsSelected == true)).FirstOrDefault();
```

Comment:
Line 54 and 55 can be in the else block because only if path in the argument is empty then you will use default path